picard/util/tags.py

```diff
@@ -278,14 +297,19 @@ def _base_description(self, tagname):
         else:
             return f"<p><em>%{name}%</em></p>{content}"
 
+    def _add_sections(self, item, include_sections):
+        # Note: format has to be translatable, for languages not using left-to-right for example
+        fmt = _("<p><strong>{title}:</strong> {values}.</p>")
+        return ''.join(item._gen_sections(fmt, include_sections))
```

Comment:
Just to help me understand, will this result in the translators having to deal with the HTML code in the translation?  I was hoping to avoid that if possible.  My thought was to translate the title and values separately, add the `<strong></strong>` wrapping around the translated title, and then substitute the translated strings into a translated format string.

Reply:
Yes, translators have to deal with HTML and for a good reason: some tags have attributes for writing direction (see https://www.w3.org/International/questions/qa-html-dir). So basically, with `gettext` we handle translations of strings, but not really formatting of text, and even if we can improve things a bit by being aware of it, that's rather tricky and testing isn't always easy for one not reading for example arabic or without having a complete translation in the target language.

Translators are usually aware enough about basic HTML and the language they translate to, so it isn't much an issue.